scene/gui/control.cpp

```diff
@@ -3054,8 +3081,10 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_global_rect"), &Control::get_global_rect);
 	ClassDB::bind_method(D_METHOD("set_focus_mode", "mode"), &Control::set_focus_mode);
 	ClassDB::bind_method(D_METHOD("get_focus_mode"), &Control::get_focus_mode);
+	ClassDB::bind_method(D_METHOD("set_focus_layer", "p_layer_id"), &Control::set_focus_layer);
+	ClassDB::bind_method(D_METHOD("get_focus_layer"), &Control::get_focus_layer);
 	ClassDB::bind_method(D_METHOD("has_focus"), &Control::has_focus);
-	ClassDB::bind_method(D_METHOD("grab_focus"), &Control::grab_focus);
+	ClassDB::bind_method(D_METHOD("grab_focus", "p_auto_set_viewport_active_focus_layer"), &Control::grab_focus, DEFVAL(false));
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("grab_focus", "auto_set_viewport_active_focus_layer"), &Control::grab_focus, DEFVAL(false));
```
`p_` prefix is only used in C++ code, this is argument name for binding.

Reply:
Ty!